org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/console/JavaStackTraceHyperlink.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2017 IBM Corporation and others.
+ * Copyright (c) 2024 IBM Corporation and others.
```

Comment:
the initial date must not be changed. the second date may be updated, but does not need to

Reply:
okay, will update it like that